staging/src/k8s.io/apimachinery/pkg/util/validation/field/errors.go

```diff
@@ -42,6 +43,15 @@ func (v *Error) Error() string {
 	return fmt.Sprintf("%s: %s", v.Field, v.ErrorBody())
 }
 
+// Is overrides the original Is method, compares error against the defined template.
+func (v *Error) Is(target error) bool {
+	t, ok := target.(*Error)
```

Comment:
I don't have a lot of experience with best practices for overridden Is implementations... is casting expected, or unwrapping using `errors.As(&Error{})`?

Reply:
The target is assumed unwrapped.

`errors.Is(err, target)` unwraps `err` sequentially looking for an error that matches the second.

Reply:
/retest